share/completions/az.fish

```diff
@@ -0,0 +1,16 @@
+function __fish_az_complete
+  set -x _ARGCOMPLETE 1
+  set -x _ARGCOMPLETE_IFS \n
+  set -x _ARGCOMPLETE_SUPPRESS_SPACE 1
+  set -x _ARGCOMPLETE_SHELL fish
+  set -x COMP_LINE (commandline -p)
```

Comment:
You typically want `commandline -pc`, to cut off the bit after the cursor.

That allows you to go back and change an argument.

Reply:
I'm not sure this makes sense since az has subcommands with subcommands which don't means swapping out one subcommand in the chain generally won't yield a valid command, but other than that it's fine. I've added the parameter